openmetadata-ui/src/main/resources/ui/playwright/utils/sidebar.ts

```diff
@@ -29,9 +29,9 @@ export const clickOnLogo = async (page: Page) => {
 export const sidebarClick = async (page: Page, id: string) => {
   const items = SIDEBAR_LIST_ITEMS[id as keyof typeof SIDEBAR_LIST_ITEMS];
   if (items) {
-    await page.hover('[data-testid="left-sidebar"]');
-    await page.waitForTimeout(300);
-    await page.click(`[data-testid="${items[0]}"]`);
+    await page.click(
+      `[data-testid="left-sidebar"] [data-menu-id*="${items[0]}"]`
+    );
```

Comment:
Don't we need to fire hover before clicking on menu?

Reply:
The behaviour varies for different tests based on the way we log in. In some cases, the left panel is open, which requires a click to open, and in some cases, it is closed, where, on hover, the sub-items will display. I'll make the behaviour uniform to have the panel closed all the times.